### ISSUE TITLE
How to update the map

### ISSUE THREAD

**Issue by washaqq** [Project-OSRM/osrm-backend#6731]
Hello,

i installed osrm-backend from ([source](https://gist.github.com/shafayeatsumit/17366c729ede9cbe36768c8bf4add569)) which i mentioned here; https://github.com/Project-OSRM/osrm-backend/issues/6725 .

My question is how can i update the map?

my assumption is to download latest map and re-run the commands in here ; https://github.com/Project-OSRM/osrm-backend/wiki/Running-OSRM#quickstart
or 
as in the ([source](https://gist.github.com/shafayeatsumit/17366c729ede9cbe36768c8bf4add569));

```
#Grab a .osm.pbf extract from 
http://download.geofabrik.de/index.html
wget http://download.geofabrik.de/europe/germany/berlin-latest.osm.pbf


#Pre-process the extract and start the HTTP server

osrm-extract berlin-latest.osm.pbf -p profiles/car.lua
osrm-contract berlin-latest.osrm
osrm-routed berlin-latest.osrm```

**Comment by nilsnolde** [comment]
Yep you got it. There’s (unfortunately) no shortcut to the full graph build pipeline under any circumstance.

**Comment by washaqq** [comment]
@nilsnolde Thank you for your reply, i see, this ok. I am planning to create cronjob to download the map and the run the ```osrm-extract, osrm-contract and osrm-routed``` codes once in a month, but do i have to do anything before, will ``extract and contract`` commands overwrite everything? do i have to clean anything before?
